config.toml

```diff
@@ -46,3 +46,8 @@ defaultContentLanguageInSubdir = false
 title = "Kamrad"
 languageName = "English"
 weight = 1
+
+# Build
+[build]
+  noJSConfigInAssets = true
```

Comment:
This is added in line 3, you could remove it  from there if adding it under build